src/content/self-hosted/administration/configuration.mdx

```diff
@@ -926,6 +926,32 @@ wildcardCertificate:
       key: "ca.crt"
 ```
 
+### user serviceaccount
```

Comment:
Is this the format we use for subkeys?

Reply:
i thought adding user to serviceaccount will determine more the key for the config, but i see a mix of titles for different keys... i will stick to `serviceaccount` because is the config of serviceaccount, and then keep the example with the key of user, WDYT? i'm not sure what would be the best here